src/exercises/exerciseTypes/audioExerciseOne/AudioExerciseOne.js

```diff
@@ -35,10 +35,14 @@ export default function AudioExerciseOne({
 
   // Timeout is set so that the page renders before the word is spoken, allowing for the user to gain focus on the page
   useEffect(() => {
+    setExerciseType(EXERCISE_TYPE);
+    api.getArticleInfo(bookmarksToStudy[0].article_id, (articleInfo) => {
+      setArticleInfo(articleInfo);
+    });
     setTimeout(() => {
       handleSpeak();
     }, 500);
-    setExerciseType(EXERCISE_TYPE);
+    setArticleInfo(articleInfo);
```

Comment:
does this line make sense here? i see it makes sense on line #40 but why have it also here?

Reply:
No, thats a mistake, I've removed it from the branch now